StreamChatClient_v3/ChatClient.swift

```diff
@@ -151,7 +151,7 @@ public final class Client<ExtraData: ExtraDataTypes> {
       assertionFailure("The URL provided in ChatClientConfig can't be `nil`.")
 
     } catch {
-      let handledError = ClientError.Unexpect(underlyingError: error)
+      let handledError = ClientError.Unexpect(error: error)
```

Comment:
I wish I've realized this name sooner... It should be `Unexpected`, since `Unexpect` is a verb